lib/src/tests/DragAndDrop.spec.tsx

```diff
@@ -0,0 +1,111 @@
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import { Suspense } from 'react';
+import VernaForm from ':/components/VernaForm';
+import VernaToolbar from ':/components/VernaToolbar';
+import VernaProvider from ':/providers/VernaProvider';
+import {
+  schemaFactory,
+  translationsFactory,
+  uiSchemaFactory,
+  widgetsFactory,
+} from ':/tests/mocks/factories';
+import ShowCaseWidgetProps from ':/types/Widgets';
+
+describe('drag and drop', () => {
+  it('should display the draggable fields', async () => {
+    function ToolbarWidget(props: ShowCaseWidgetProps) {
+      return <span>{props.widgetName}</span>;
+    }
+
+    render(
+      <VernaToolbar>
+        <ToolbarWidget type="string" widgetName="textwidget" />
+        <ToolbarWidget type="number" widgetName="numberwidget" />
+      </VernaToolbar>,
+    );
+
+    const $numberWidget = screen.getByText('numberwidget').parentElement;
+    expect($numberWidget);
+    expect($numberWidget!.draggable).toBeTruthy();
```

Comment:
I do not like `toBeTruthy` method as it could leads to false positive.
```js
expect('false').toBeTruthy(); // true
```

```suggestion
    expect($numberWidget!.draggable).toBe(true);
```

Reply:
Oh i didn't think about it this way, indeed way better practice to do it like this ! thanks !